### PR TITLE
Documentation and bundles fixes

### PR DESCRIPTION
**Pull request by richafrank**
Before:
image

After:
image

Before:
image

After:
image

### CODE REVIEW

File: zipline/data/bundles/core.py [hunk 1]
```diff
@@ -586,8 +586,8 @@ def clean(name,
             def should_clean(name):
                 dt = from_bundle_ingest_dirname(name)
                 return (
-                    (before is not None and dt < before) or
-                    (after is not None and dt > after)
+                    (before is None or dt < before) and
+                    (after is None or dt > after)
```
Comment (richafrank) [inline zipline/data/bundles/core.py:590]: @llllllllll What is the intended way to clean out all the ingestions for a bundle? I tried with no args, but nothing happened. However, my change looks to fail the relevant test...

Comment (llllllllll) [reply zipline/data/bundles/core.py:590]: You need to pass either `--before`, `--after` or `--keep-last` to specify which ingests you would like to delete.

Comment (llllllllll) [inline zipline/data/bundles/core.py:590]: why invert all of these conditions?

File: docs/source/conf.py [hunk 2]
```diff
@@ -72,7 +72,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['.static']
+html_static_path = []
```
Comment (llllllllll) [inline docs/source/conf.py:75]: do we not have any images or anything?

File: zipline/__main__.py [hunk 3]
```diff
@@ -317,13 +320,6 @@ def run(ctx,
         benchmark_spec=benchmark_spec,
     )
 
-    if output == '-':
```
Comment (llllllllll) [inline zipline/__main__.py:320]: why can we remove this logic?

File: zipline/__main__.py [hunk 4]
```diff
@@ -112,6 +112,9 @@ def _(*args, **kwargs):
     return d
 
 
+DEFAULT_BUNDLE = 'quantopian-quandl'
```
Comment (llllllllll) [inline zipline/__main__.py:115]: :+1:

File: zipline/data/bundles/core.py (hunk 1 above)
Comment (richafrank) [reply zipline/data/bundles/core.py:590]: Ah, ok, that was my confusion. I expected that passing none of them would delete everything.

Comment (richafrank) [reply zipline/data/bundles/core.py:590]: This was my attempt to fix the noop `clean` commands (when I passed no args). I'll instead raise `BadClean` in that case.

File: docs/source/conf.py (hunk 2 above)
Comment (richafrank) [reply docs/source/conf.py:75]: Not there at least. We have 2 pngs in docs/tutorial_files.

```
Running Sphinx v3.1.2
making output directory... done
WARNING: html_static_path entry '.static' does not exist
```

File: zipline/__main__.py (hunk 3 above)
Comment (richafrank) [reply zipline/__main__.py:320]: The final lines of `_run` (called above) are:
```python
    if output == '-':
        click.echo(str(perf))
    elif output != os.devnull:  # make the zipline magic not write any data
        perf.to_pickle(output)

    return perf
```

File: zipline/data/bundles/core.py [hunk 5]
```diff
@@ -578,6 +578,9 @@ def clean(name,
             if e.errno != errno.ENOENT:
                 raise
             raise UnknownBundle(name)
+
+        if before is after is keep_last is None:
```
Comment (llllllllll) [inline zipline/data/bundles/core.py:582]: :+1:

**Pull request merged**